src/trace.c

```diff
@@ -1007,6 +1007,12 @@ trace_add_path (const void *parent,
 	static NihHash *inode_hash = NULL;
 	nih_local char *inode_key = NULL;
 
+	/* Used when the file to be included turns out
+	 * to be a symlink.
+	 * stores a resolved path from the symlink.
+	 */
+	static char resolved_pathname[PATH_MAX+1] = {0};
```

Comment:
I don't think we need `= {0}` since we are doing `memset` when we actually found a symlink

Reply:
Done.